.github/CODEOWNERS

```diff
@@ -3,22 +3,22 @@
 /crates/sui-keys/ @joyqvq @patrickkuo
 /crates/sui-protocol-config @mystenmark
 
-*.md @randall-Mysten @ronny-mysten
+*.md @ronny-mysten
 # Omit auto-generated changelogs and changesets from docs review:
 /.changeset/*.md
 CHANGELOG.md
 
-# When snapshot changes, most likely we will need to recreate the genesis blob.
-/crates/sui-config/tests/snapshot_tests.rs @tharbert
-
 # When native functions change, so we can evaluate cost impact
-/crates/sui-framework/src/natives/ @oxade
+/crates/sui-framework/src/natives/ @dariorussi
```

Comment:
tagging @dariorussi here - I can remove this entirely if it's no longer relevant